play/PlayerRules/CharacterCreation/04_01_Weapons.txt

```diff
@@ -253,32 +253,136 @@ Minimum Strength 4, Reload DC 25% unless otherwise noted. Reload time 2 Actions.
+	4 out to 20m
+	5 out to 35m
+Burst Miss Chance:		+3 - 1 for every 6 Dex (starting at 5)
+Automatic Miss Chance:	+4 - 1 for every 5 Dex
+Mag size: 33 rounds of 8.0x20mm. $18 per mag
+Reload DC: 			30
+Fire modes:			Semi, burst and auto
+Burst; Auto fire rate:		3; 6
+Armor pierce level: 		1
+Reflex Modifier: 			+10
+Suppression Level:		Medium
+
+----ECSC Mk9 PDW 	$1027
+Damage:	45
+Ranged Miss Chance:
+	4 out to 10m
+	5 out to 20m
+	6 out to 40m
+Automatic Miss Chance:	+4 - 1 for every 5 Dex
+Mag size: 25 rounds of 9.0x16mm. $12 per mag
+Reload DC: 			30%
+Fire modes:			Semi, auto
+Auto fire rate:			8
+Armor pierce level: 		0
+Reflex Modifier: 			+10
+Suppression Level: 		Medium
+
+=====	LEVEL 5 REQUIREMENT WEAPONS	=====
+
+----G&W XMP	$1536
+Damage:	55
+Ranged Miss Chance:
+	4 out to 10m
+	5 out to 20m
+	6 out to 40m
+Burst Miss Chance:		+4 - 1 for every 4 Dex
+Automatic Miss Chance:	+6 - 1 for every 4 Dex
+Mag Size:	30 rounds of 10.0x25mm. $19 per mag
+Reload DC:				30%
+Fire Modes:			Semi, Burst, Auto
+Burst Fire Rate:			3
+Auto fire rate:			6
+Armor Pierce Level:		2
+Reflex Modifier:			+10
+Suppression Level:		Medium
+
+---- Anvil Arms AR-34 PDW	$1356
+Damage:	45
+Ranged Miss Chance:
+	3 out to 10m
+	4 out to 25m
+	5 out to 45m
+Automatic Miss Chance:	+3 - 1 for every 8 Dex
+Mag Size:	15 rounds of 5.00x37mm. $7 per mag
+Reload DC:				30%
+Fire Modes:			Semi, Auto
+Auto fire rate:			6
+Armor Pierce Level:		3
+Reflex Modifier:			10
+Suppression Level:		Medium
+
+----Flexcorp LS55	$2183
+Damage:	35
+Ranged Miss Chance:
+	3 out to 15m
+	4 out to 40m
+	5 out to 50m
+Automatic Miss Chance:	+3 - 1 for every 8 Dex
+Mag Size:	40 rounds of 10kW (7x20mm). $23 per mag
+Reload DC:				35%
+Fire Modes:			Semi, Auto
+Auto fire rate:			8
+Armor Pierce Level:		n/a
+Reflex Modifier:			10
+Suppression Level:		Medium
+
+=====	LEVEL 10 REQUIREMENT WEAPONS	=====
+
+---- Mathews "Bolter" 470B	$2389
+Damage:	60, 1x bleed on hit
+Ranged Miss Chance:
+	3 out to 10m
+	4 out to 35m
+	5 out to 45m
+Automatic Miss Chance:	+4 - 1 for every 5 Dex
+Mag Size:	20rounds of 4x70mm. $13 per mag
+Reload DC:				30%
+Fire Modes:			Semi, Auto
+Auto fire rate:			4
+Armor Pierce Level:		2
+Reflex Modifier:			+5
+Suppression Level:		Medium
+
+----Frontier PL33G 	$2245
+Damage:	55, 1x plasma burn on hit
+Ranged Miss Chance:
+	3 out to 10m
+	4 out to 30m
+	6 out to 40m
+Burst Miss Chance:		+2 - 1 for every 8 Dex;
+Automatic Miss Chance:	+5 - 1 for every 5 Dex
+Mag Size:	20rounds of Argon-9 9x22mm. $16 per mag
+Reload DC:				40%
+Fire Modes:			Semi, Burst, Auto
+Burst Fire Rate:			3
+Auto fire rate:			6
+Armor Pierce Level:		2
+Reflex Modifier:			+10
+Suppression Level:		Medium
+
+---- Flexcorp LS75	$3670
+Damage:	45
+Ranged Miss Chance:
+	2 out to 15m
+	3 out to 40m
+	4 out to 50m
+Automatic Miss Chance:	+4 - 1 for every 5 Dex
+Mag Size:	30 rounds of 10kW 8x22mm. $22 per mag
+Reload DC:				35%
+Fire Modes:			Semi, Auto
+Auto fire rate:			6
+Armor Pierce Level:		N/A
+Reflex Modifier:			+10

```

Comment:
Please view the file on github to see all the tab-spacing problems. The spacing is all over the place on most of the weapons.